tests/PHPUnit/Integration/TrackerTest.php

```diff
@@ -357,6 +359,30 @@ public function test_main_shouldPostEndEvent_EvenIfThereIsAnException()
         $this->assertTrue($called);
     }
 
+    public function test_archiveInvalidation_differentServerAndWebsiteTimezones()
+    {
+        // Server timezone is UTC
+        ini_set('date.timezone', 'America/New_York');
```

Comment:
Should this be set down in a tearDown method to make sure the timezone isn't different for other tests?

Reply:
👍  in `setUp` we would maybe also need to remember the initial timezone and then restore the original timezone in `tearDown`?